src/BenchmarkDotNet/Engines/ThreadingStats.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Threading;
+
+namespace BenchmarkDotNet.Engines
+{
+    public struct ThreadingStats
+    {
+        internal const string ResultsLinePrefix = "Threading: ";
+
+        // BDN targets .NET Standard 2.0, these properties are not part of it
```

Comment:
`..these properties are part of .Net Core 3.0+`. I think it will be more understandable for future readers.